build-docker-image/README.md

```diff
@@ -17,6 +25,7 @@ Optionally (recommended) scan the image for vulnerabilities using [Snyk](https:/
 - `context`: Path used as file context for Docker. If not set, the git repository is cloned using the same git reference as the workflow and used as context.
 - `target`: The target stage to build of a multi-stage build
 - `docker-repository`: Repository name in the container registry. e.g. ghcr.io/dfe-digital/register-trainee-teachers. Defaults to current Github repository name.
+- `max-cache`: Set to true to use maximum cache level when reuse-cache is set. Defaults to minimum (false)
```

Comment:
Can we add a warning that a refresh workflow should be in place

Reply:
added to readme